<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissLoanQuery</class>
 <widget class="QWidget" name="MissLoanQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>61</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MissLoanQuery</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblDesc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>外借查询:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cboxCondition">
     <item>
      <property name="text">
       <string>产品型号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>序列号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>外借日期</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>外借人</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>最终用户</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>预计归还日期</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtValue"/>
   </item>
   <item>
    <widget class="QComboBox" name="cboxOperator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>&lt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cBoxState">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>外借中</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已归还</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>已销售</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>全部</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnQuery">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cboxCondition</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MissLoanQuery</receiver>
   <slot>s_conditionChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnQuery</sender>
   <signal>clicked()</signal>
   <receiver>MissLoanQuery</receiver>
   <slot>s_btnQuery()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtValue</sender>
   <signal>returnPressed()</signal>
   <receiver>btnQuery</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_conditionChange(int)</slot>
  <slot>s_btnQuery()</slot>
 </slots>
</ui>
